stlab/config.hpp.in

```diff
@@ -74,9 +74,11 @@
         #define STLAB_CPP_VERSION_PRIVATE() 14
     #elif __cplusplus == 201703L
         #define STLAB_CPP_VERSION_PRIVATE() 17
-    #else
-        //#warning Unknown version of C++, assuming C++20.
+    #elif __cplusplus == 202002L
```

Comment:
Where are you getting 202002L from?

Reply:
https://en.cppreference.com/w/cpp/preprocessor/replace#Predefined_macros